packages/scripts/src/flux-emulator/ReferenceContractConfig.ts

```diff
@@ -210,9 +214,24 @@ export const addAdapterToConfig = (
 ): ReferenceContractConfig[] => {
   for (const config of masterConfig) {
     for (const node of config.nodes) {
-      if (node.dataProviders.includes(adapterName)) {
+      for (const dataProvider of node.dataProviders) {
+        const [name, params] = dataProvider.split('?')
+        if (name !== adapterName && (!rddAliases[name] || rddAliases[name] !== adapterName))
+          continue
+
```

Comment:
Can you add examples of these new operations in the tests for this? https://github.com/smartcontractkit/external-adapters-js/blob/e19377822b8d8d4b2f82f27928ed2fdaf1dd25ee/packages/scripts/src/flux-emulator/ReferenceContractConfig.test.ts

Reply:
Updated!